definitions/npm/react-apollo_v2.x.x/flow_v0.58.x-/react-apollo_v2.x.x.js

```diff
@@ -7,10 +7,10 @@ import type {
   ApolloQueryResult,
   ApolloError,
   FetchPolicy,
-  FetchMoreOptions,
+  FetchMoreOptions as FetchMoreOptionsAC,
   UpdateQueryOptions,
-  FetchMoreQueryOptions,
-  SubscribeToMoreOptions,
```

Comment:
I'm assuming you're already aware of this, but just to be sure, all of these imports from apollo-client are just resolving to any (both because this import is outside of declare module and because a definition doesn't currently exist). 

Started on a PR to add a definition for apollo-client: https://github.com/flowtype/flow-typed/pull/2172

Reply:
Yes, I know 👍

Reply:
FAQ for reference: https://github.com/flowtype/flow-typed/wiki/FAQs#how-do-i-build-type-definitions-that-depend-on-other-definitions.